databroker/broker.py

```diff
@@ -708,6 +708,62 @@ def process(self, headers, func, fields=None, fill=False):
 
     get_fields = staticmethod(get_fields)  # for convenience
 
+    def export(self, headers, db, new_root=None):
+        """
+        Export a list of headers.
+
+        .. warning::
+
+            This does not yet support copying external files.
+
+        Parameters:
+        -----------
+        headers : databroker.header
+            one or more headers that are going to be exported
+        db : databroker.Broker
+            an instance of databroker.Broker class, which has
+            filestore (fs) and metadatastore (mds) attributes
+            that will be the target to export info
+        new_root : str
+            optional. root directory of files that are going to
+            be exported
+        """
+        try:
+            headers.items()
+        except AttributeError:
+            pass
+        else:
+            headers = [headers]
+        for header in headers:
+            # insert mds
+            db.mds.insert_run_start(**_clean_doct(header['start']))
```

Comment:
Because a doct has a _name attribute hanging off of it that gets unpacked in insert_run_start.